### PR TITLE
Use new Onco-Query-Language.md for OQL page

### PR DESCRIPTION
**Pull request by adamabeshouse**
image

### CODE REVIEW

File: portal/src/main/webapp/onco_query_lang_desc.jsp [hunk 1]
```diff
@@ -30,26 +30,52 @@
  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (adamabeshouse) [inline portal/src/main/webapp/onco_query_lang_desc.jsp]: this file is basically `Ctrl+C` `Ctrl+V` from news.jsp, with the page layout and title changed.

**Comment by jjgao** [comment]
@adamabeshouse is there any way to properly format the table in `OQL Keywords` section?

**Comment by adamabeshouse** [comment]
@jjgao @tmazor I downloaded a github-flavored markdown styles (gfm.css) and had to update our markdown-to-html converter library, please see updated screenshot/heroku

**Comment by adamabeshouse** [comment]
I'm seeing some weird errors that strangely disappear when I use my local frontend instance and localdev:

image

**Comment by tmazor** [comment]
It looks much better!

There are three links towards the end of the page that are currently just text. @adamabeshouse can you make them links?
image

**Comment by adamabeshouse** [comment]
@tmazor to that end I just modified the markdown file, it should work now when that is merged (though it won't work on heroku because its not yet merged)

**Comment by adamabeshouse** [comment]
@jjgao can we merge this?

**Comment by jjgao** [comment]
@adamabeshouse I see some strange characters in Keyword and Default columns.

image

**Comment by adamabeshouse** [comment]
I was seeing that too but I think it may go away in production, because for some reason it disappeared when I used localdev. Maybe thats misguided..

**Pull request merged**